.github/workflows/tests_skipper.yml

```diff
@@ -12,6 +12,7 @@ on:
       - "**.py"
```

Comment:
Can we keep the same ordering as `tests.yml`? Makes it easier to read.

Reply:
Agree